client/my-sites/customer-home/main.jsx

```diff
@@ -60,27 +60,38 @@ const Home = ( {
 		);
 	}
 
+	const header = (
+		<div className="customer-home__heading">
+			<FormattedHeader
+				headerText={ translate( 'My Home' ) }
+				subHeaderText={ translate( 'Your home base for posting, editing, and growing your site.' ) }
+				align="left"
+			/>
+			<div className="customer-home__view-site-button">
+				<Button href={ site.URL } onClick={ trackViewSiteAction }>
+					{ translate( 'View site' ) }
+				</Button>
+			</div>
+		</div>
+	);
+
 	return (
 		<Main className="customer-home__main is-wide-layout">
 			<PageViewTracker path={ `/home/:site` } title={ translate( 'My Home' ) } />
 			<DocumentHead title={ translate( 'My Home' ) } />
 			{ siteId && <QuerySiteChecklist siteId={ siteId } /> }
 			{ siteId && <QueryHomeLayout siteId={ siteId } /> }
 			<SidebarNavigation />
-			<div className="customer-home__heading">
-				<FormattedHeader
-					headerText={ translate( 'My Home' ) }
-					subHeaderText={ translate(
-						'Your home base for posting, editing, and growing your site.'
-					) }
-					align="left"
-				/>
-				<div className="customer-home__view-site-button">
-					<Button href={ site.URL } onClick={ trackViewSiteAction }>
-						{ translate( 'View site' ) }
-					</Button>
-				</div>
-			</div>
+			<Experiment name="user_home_test" event={ 'calypso_saw_a_a_test' }>
```

Comment:
Consider changing the event name to be `calypso_aatest_start` to follow the naming pattern of the existing `calypso_abtest_start` event.
```suggestion
			<Experiment name="user_home_test" event="calypso_aatest_start">>
```

Reply:
I'd rather not anyone ever use this event again.

Reply:
But in all seriousness, it has very different semantics than the abtest_start event.

Reply:
That's fair. Yeah, if we change the name, we should go all the way and also match the eventprops like `abtest_name` and `abtest_variation`.

Reply:
It's also just to make things easier for cases where there's already not an exposure event readily available. AFAIK, the event props isn't used so much (unless specified in the experiment definition). In fact, an event is not required at all, I just did it so we could enable all the analysis types and see how they compare. If anything, I suggest making the event name more "ridiculous" just to make it obvious that the event name doesn't matter.

Reply:
Agreed with not using `calypso_aatest_start`, but `calypso_saw_a_a_test` doesn't match [the Tracks naming conventions](https://fieldguide.automattic.com/tracks/tracks-for-developers-product-teams/tracks-naming-conventions/). Let's go with something more compliant, like `calypso_customer_home_aa_experiment_exposed`.

Reply:
Scrap that. We shouldn't be firing an event at all: https://github.com/Automattic/wp-calypso/pull/42093#pullrequestreview-411473293. It isn't the responsibility of the experimentation client and it encourages bad practices and event inflation.